### PR TITLE
added some questions in adder.md

### PR DESCRIPTION
**Pull request by ayan-biswas0412**
A meaningful title for PR, not like: made changes to xyz.md

Fixes #505 

#### Changes done:
- [x] added some questions in adder.md and given some multi option correct questions also

#### Screenshots
image

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by ayan-biswas0412** [comment]
@Shivansh2407 please review

File: docs/comb-msi/adders.md [hunk 1]
```diff
@@ -169,3 +169,39 @@ If A > B Cout = 0 and the result of binary form (A-B) then Cout = 1 and the resu
...
+
+{:.quiz}
+
+1. Consider a half adder circuit - for which of the following inputs A,B the sum (S) output is same ?
+	1. 0,0
+	* 0,1
+	* 1,0
+	1. 1,1
+
+2. Using only which of the following gates in minimum number Half Adder circuit can be implemented?
```
Comment (manjotsidhu) [inline docs/comb-msi/adders.md:182]: Space before "?" is required.
```suggestion
2. Using only which of the following gates in minimum number Half Adder circuit can be implemented ?
```

**Review by manjotsidhu — CHANGES_REQUESTED** [review]

**Comment by ayan-biswas0412** [comment]
@manjotsidhu please review I have done the changes

**Review by manjotsidhu — APPROVED** [review]

**Review by Shivansh2407 — APPROVED** [review]
Great Work @ayan-biswas0412 🎉

**Pull request merged**